src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -32,51 +38,108 @@ type Settings struct {
 	AutoReport   bool
 	AutoLaunch   bool
 	ProxyAll     bool
+	InstanceID   string
+
+	sync.RWMutex
 }
 
-func Configure(cfg *config.Config, version, revisionDate string, buildDate string) {
+// Load loads the initial settings at startup, either from disk or using defaults.
+func Load(version, revisionDate, buildDate string) {
+	// Create default settings that may or may not be overridden from an existing file
+	// on disk.
+	settings = &Settings{
+		Version:      version,
```

Comment:
Oh I see, they are shown in Lantern UI. But by persisting them to file, they will not change across upgrades. Maybe we can set it after loading from settings file, or change lantern-ui assets directly for each version.